.gitignore

```
@@ -4,3 +4,4 @@
 # Env files
 .env
 .data/
+.*keys*
```

[question] What is this file?

✨  Bonus suggestion ✨ 
```suggestion
.*keys*

```